state/machine_internal_test.go

```diff
@@ -0,0 +1,57 @@
+package state
+
+import (
+	gc "gopkg.in/check.v1"
+	"gopkg.in/mgo.v2/bson"
+	"gopkg.in/mgo.v2/txn"
+)
+
+type MachineInternalSuite struct {
```

Comment:
I would think we would want at least testing.IsolationSuite, in case any of the objects end up calling a global logger.Tracef/Debugf that we get that captured into the test suite, instead of sent as stipple to stderr.